### PR TITLE
[Fix IntelliJ tests] Prevent class naming conflicts

### PR DESCRIPTION
**Pull request by c-harding**
Classes and package objects can’t share names between main, test and it.

### CODE REVIEW

**Comment by c-harding** [comment]
Turns out reducing the indent on an entire file breaks git

File: src/it/scala/temple/containers/DockerPostgresService.scala [hunk 1]
```diff
@@ -8,6 +8,23 @@ import temple.containers.DockerPostgresService.PostgresReadyChecker
 import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration._
 
+/** DockerPostgresService configures a docker container for Postgres */
+trait DockerPostgresService extends DockerKit {
```
Comment (c-harding) [inline src/it/scala/temple/containers/DockerPostgresService.scala:12]: Traits need to be declared above their companion object

**Review by jaylees14 — APPROVED** [review]
👍

**Review by Lewiky — APPROVED** [review]
LGTM

**Pull request merged**